test/acceptance/wdio.conf.js

```diff
@@ -1,7 +1,7 @@
 const { ReportAggregator, HtmlReporter } = require('@rpii/wdio-html-reporter')
 const log4js = require('@log4js-node/log4js-api')
 const logger = log4js.getLogger('default')
-const envRoot = (process.env.TEST_ENVIRONMENT_ROOT_URL || 'https://ffc-demo.ffc-dev.azure.defra.cloud')
+const envRoot = (process.env.TEST_ENVIRONMENT_ROOT_URL || 'http://ffc-demo.ffc.snd.azure.defra.cloud')
```

Comment:
I'm not sure this should be the default if the env var isn't available. An alternative would be to have `http://host.docker.internal:3000` so it works locally (on Windows on macOS) without an env var set. However, I'm not convinced. What do you think?

Reply:
That's a good call, especially given the default that exists within the Jenkins Pipeline is the same, I'm swayed.